### PR TITLE
Add searchbar to list page

### PR DESCRIPTION
**Pull request by Kshitij-Kumar123**
select different universities to add in your list and automatically add data. 
currently gets a hardcoded variable for data, which will be replaced

### CODE REVIEW

**Comment by Kshitij-Kumar123** [comment]
Do not merge this branch. This branch is work in progress

**Comment by Kshitij-Kumar123** [comment]
focusing on functionality right now. css changes for the horizontal dashboard cards to be done in a future pr

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -21,13 +23,21 @@ export default function Navbar(props) {
...
-        <Button variant="link" style={{color: 'white'}} onClick={handleLogOut}>
-          Sign Out
-        </Button>
-        <FaUserCircle style={{ fontSize: "2em" }} />
+        <Dropdown>
+          <Dropdown.Toggle variant="danger" style={{backgroundColor:'#f06b6b', border: 'none'}}>
+            <FaUserCircle style={{ fontSize: "2em" }} />
+          </Dropdown.Toggle>
+          <Dropdown.Menu>
+            <Dropdown.Item onClick={handleLogOut}>Sign Out</Dropdown.Item>
```
Comment (valamuri2020) [inline src/components/Navbar/Navbar.jsx]: put signout below user email

File: src/components/list/listCard.jsx [hunk 2]
```diff
@@ -1,83 +1,136 @@
...
+          <Heading>AVG SAT</Heading>
+          <Input value={selectedCollege?.["SAT_AVG_ALL"]} onChange={(e) => handleChange(e.target.value, 'SAT_AVG_ALL')}/>
+        </FormGroup>
+        <FormGroup>
+          <Heading>AVG ACT</Heading>
+          <Input value={selectedCollege?.["ACT_AVG"]} onChange={(e) => handleChange(e.target.value, 'ACT_AVG')}/>
+        </FormGroup>
+        <FormGroup>
+          <Heading>Acceptance %</Heading>
+          <Input value={selectedCollege?.["ACTENMID"]} onChange={(e) => handleChange(e.target.value, 'ACTENMID')}/>
```
Comment (valamuri2020) [inline src/components/list/listCard.jsx]: please change this to acceptance data from ACTENMID data

**Review by valamuri2020 — CHANGES_REQUESTED** [review]

**Review by valamuri2020 — APPROVED** [review]

**Pull request merged**